overlord/ifacestate/ifacemgr.go

```diff
@@ -273,6 +279,9 @@ func (m *InterfaceManager) Ensure() error {
 // if running.
 func (m *InterfaceManager) Stop() {
 	m.stopUDevMon()
+	// XXX: The state lock is *not* held during Stop(). If it were, we'd need
+	// to unlock state and re-lock it after the call to
+	// interfacesRequestsManagerStop, so that it can record notices if needed.
```

Comment:
@pedronis please confirm that this is the case, from what I could tell this is true but not certain

Reply:
yes, afaik we don't hold the state lock when entering any of the manager methods driven by StateEngine, you could probably replace the XXX with a comment about that